django/contrib/staticfiles/storage.py

```diff
@@ -73,6 +73,13 @@ class HashedFilesMixin:
                 r"""(?P<matched>import\(["'](?P<url>.*?)["']\))""",
                 """import("%(url)s")""",
             ),
+            (
+                (
+                    r"""(?P<matched>(?P<import>import\*as\s\w+)"""
+                    r"""\s*from\s*['"](?P<url>[./].*?)['"]\s*;)"""
+                ),
+                """%(import)s from "%(url)s";""",
+            ),
```

Comment:
This regex does not work. It breaks the tests. It gives this error `ValueError: The file 'absolute_root.5586327fe78c.js' could not be found with <staticfiles_tests.test_storage.JSModuleImportAggregationManifestStorage object at 0x7c3a413a3cb0>.` Is  the changes needed for extra tests that we added? because I tested it with added tests they work with this regex `
r"""(?P<matched>(?P<import>import\*\s?as\s\w+)"""
r"""\s*from\s*['"](?P<url>[./].*?)['"]\s*;)"""
`

Reply:
No I'm just questioning whether this case can be incorporated into an existing regex rather than needing a new regex

Reply:
For the tests you suggested, I changed the same regex to fit those as well.
If there are more test cases I can test them as well.

Reply:
`(
                    r"""(?P<matched>import(?s:(?P<import>[\s\{]?.*?))"""
                    r"""\s*from\s*['"](?P<url>[./].*?)["']\s*;)"""
),
"""import%(import)s from "%(url)s";""",`
 This regex catches all the test cases. But for some reason, it breaks one export test 
![Screenshot from 2024-08-06 01-28-20](https://github.com/user-attachments/assets/c1b56dbd-0836-4eb2-9a17-483b40ace225)

Reply:
old regex is updated to work with new tests also